python_modules/dagster/dagster/core/definitions/asset_layer.py

```diff
@@ -415,10 +438,14 @@ def from_graph_and_assets_node_mapping(
         asset_key_by_input: Dict[NodeInputHandle, AssetKey] = {}
         asset_info_by_output: Dict[NodeOutputHandle, AssetOutputInfo] = {}
         asset_deps: Dict[AssetKey, AbstractSet[AssetKey]] = {}
+        asset_io_managers: Dict[AssetKey, str] = {
```

Comment:
extreme nitpick: naming this `io_manager_keys_by_asset_key` would make it more clear  where it's going to be used